lib/core/public/run-virtual-rule.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Run a rule in a non-browser environment
+ * @param {String} ruleId  Id of the rule
+ * @param {VirtualNode} vNode  The virtual node to run the rule against
+ * @param {Object} options  (optional) Set of options passed into rules or checks
+ * @return {Object} axe results for the rule run
+ */
+axe.runVirtualRule = function(ruleId, vNode, options) {
+	'use strict';
+	let rule = axe._audit.rules.find(rule => rule.id === ruleId);
+
+	if (!rule) {
+		return;
```

Comment:
I think we should `throw` here rather than failing silently.

Consider the following:

```js
const results = axe.runVirtualRule('snuffaluffagus', someVirtualNode)
console.log(results.violations) // will throw a very unhelpful error
```

However, if we added a `throw new Error('Invalid rule or something helpful here')`, then users would immediately know what they need to do to fix their issue.

Reply:
Good point. Will make it throw instead